package.json

```diff
@@ -29,6 +36,8 @@
     },
     "peerDependencies": {
         "@dhis2/app-runtime": "^2.0.4",
-        "d2": "^31.8.1"
+        "d2": "^31.8.1",
+        "react": "^16.13.1",
+        "react-dom": "^16.13.1"
```

Comment:
These should be `^16.8`, we don't rely on any later features of React

Reply:
ok, I'll update.